.github/workflows/kfp-kubernetes-execution-tests.yml

```diff
@@ -0,0 +1,72 @@
+name: k8s execution tests
+
+on:
+  push:
+    branches: [master]
+
+  pull_request:
+    paths:
+      - '.github/workflows/kfp-kubernetes-execution-tests.yml'
+      - 'sdk/python/**'
+      - 'api/v2alpha1/**'
+      - 'kubernetes_platform/**'
+
+jobs:
+  kfp-kubernetes-execution-tests:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Create KFP cluster
+        uses: ./.github/actions/kfp-cluster
+
+      - name: Forward API port
+        run: ./scripts/deploy/github/forward-port.sh "kubeflow" "ml-pipeline" 8888 8888
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+
+      - name: apt-get update
+        run: sudo apt-get update
+
+      - name: Install protobuf-compiler
+        run: sudo apt-get install protobuf-compiler -y
+
+      - name: Install setuptools
```

Comment:
We need to create scripts and call them from GitHub Actions (GHA). This way, we can run the test suite independently of GitHub. Many vendors depend on their infrastructure to certify releases. Additionally, this approach enables component reuse within GHA.

Reply:
Agreed. We can take that approach in all tests.
Can we do that in a follow-up effort after the migration to GHA? It's quite important to have the tests running in GHA ASAP.

cc @DharmitD @HumairAK

Reply:
Ok!